### PR TITLE
:white_check_mark: Update failing tests

### PR DESCRIPTION
**Pull request by JMicheli**
Several doctests were failing and I got them passing.

### CODE REVIEW

**Comment by JMicheli** [comment]
I ended up doing this after reviewing #281 while looking through the current testing. I'd be happy to continue working on that issue and making additions here prior to merging.

Let me know if there are particular parts of the project that need test coverage.

**Comment by aaronleopold** [comment]
Thank you so much, I appreciate you taking the time to fix these!

> I'd be happy to continue working on that issue and making additions here prior to merging.
>
> Let me know if there are particular parts of the project that need test coverage.

I responded on Discord this AM, but I think a good candidate for adding tests and/or general clean up would be `core/src/opds`. It is the oldest part of the codebase, but I think asserting the generated XML for various entities could be beneficial. 

I'll leave it up to you if: a). you would like to tackle it and b). if you would rather put it all in this PR or separate it out.

File: core/src/context.rs
Comment (aaronleopold) [inline core/src/context.rs:127]: Not a LOTR fan? 😉

**Review by aaronleopold — APPROVED** [review]
LGTM, thanks again! As discussed offline, I'll merge this once CI passes

**Comment by aaronleopold** [comment]
Looks like cargo is complaining about format. I'll be able to correct it tomorrow if you can't get to it

**Comment by aaronleopold** [comment]
> Looks like cargo is complaining about format. I'll be able to correct it tomorrow if you can't get to it

For future reference, clippy --fix won't correct warnings unless you run:

```bash
__CARGO_FIX_YOLO=1 cargo clippy --fix
```

lol

**Comment by JMicheli** [comment]
I was wondering why they didn't get caught by the auto formatting.

Thanks for handling, I've spent like 8 hours outside the office in the past three days so I didn't have a chance to fix the issue.

**Comment by aaronleopold** [comment]
Yeah it definitely made me double take for a second when it just wasn't working.

>  Thanks for handling, I've spent like 8 hours outside the office in the past three days

No problem at all - that's **rough**! I hope you have some solid rest and recovery over the weekend or sometime soon

**Pull request merged**
